src/lib/is-installed.ts

```diff
@@ -2,39 +2,114 @@ import { app } from 'electron';
 import fs from 'fs';
 import path from 'path';
 
-let appPaths: Record< keyof InstalledApps, string[] >;
+type InstalledApp =
+	| 'vscode'
+	| 'phpstorm'
+	| 'cursor'
+	| 'windsurf'
+	| 'nova'
+	| 'webstorm'
+	| 'sublime'
+	| 'atom';
 
-if ( process.platform === 'darwin' ) {
-	// Get both system and user Applications directories
-	const systemApplications = '/Applications';
-	const userApplications = path.join( app.getPath( 'home' ), 'Applications' );
+type PlatformPaths = {
+	[ K in InstalledApp ]: string[];
+};
 
-	appPaths = {
+// Define installation paths for each IDE by platform
+const installationPaths: Record< string, PlatformPaths > = {
+	darwin: {
+		vscode: [ 'Visual Studio Code.app' ],
+		phpstorm: [ 'PhpStorm.app' ],
+		cursor: [ 'Cursor.app' ],
+		windsurf: [ 'Windsurf.app' ],
+		nova: [ 'Nova.app' ],
+		webstorm: [ 'WebStorm.app' ],
+		sublime: [ 'Sublime Text.app' ],
+		atom: [ 'Atom.app' ],
+	},
+	win32: {
 		vscode: [
-			path.join( systemApplications, 'Visual Studio Code.app' ),
-			path.join( userApplications, 'Visual Studio Code.app' ),
+			'C:\\Program Files\\Microsoft VS Code',
```

Comment:
Can we use variables from electron app instead of hardcoding path and disk letter?

Reply:
That's a good point, updated it in a78fcca